projects/orbit/project.yaml

```diff
@@ -0,0 +1,10 @@
+homepage: "https://github.com/google/orbit"
+language: c++
+primary_contact: "hebecker@google.com"
+auto_ccs :
+  - "orbitprofiler-eng+fuzztests@google.com"
+
+fuzzing_engines:
```

Comment:
Do you know that orbit will only work with libFuzzer+ASAN? I think it will be worth checking if it works with other sanitizers/fuzzers.

Reply:
I haven't tried other fuzzers and sanitizers yet, but that's on my TODO list.